Makefile

```diff
@@ -5,15 +5,19 @@ watch: ## Start a file watcher to run tests on change. (requires: watchexec)
 .PHONY: all
 all: lint test build ## test -> lint -> build
 
+.PHONY: verify
+verify: ## Verify dependecies 
+	go mod verify
+
 .PHONY: deps
-deps: ## Setup or Update linters
-	go get -u github.com/kisielk/errcheck
-	go get -u golang.org/x/lint/golint
-	go get -u github.com/securego/gosec/cmd/gosec
+## Verify and then Setup or Update linters
+deps: verify 
+	cd && \
+	go get -u gotest.tools/gotestsum && \
+	go get -u github.com/kisielk/errcheck && \
+	go get -u golang.org/x/lint/golint && \
+	go get -u github.com/securego/gosec/cmd/gosec && \
 	go get -u honnef.co/go/tools/cmd/staticcheck
```

Comment:
I don't like the `&& \`, can we leverage this answer?
https://stackoverflow.com/a/30590240/5272101
If that works, deps could just fire `cd`, and then as usual just `go get` everything